### PR TITLE
JOSS submision

### PR DESCRIPTION
**Pull request by htwangtw**
- JOSS submission essentials
- skip code spell on paper because it doesn't handle french

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  - functional connectivity
+authors:
+  - name: Hao-Ting Wang
+    corresponding: true
+    orcid: 0000-0003-4078-2038
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Rémi Gau
+  - name: Natasha Clarke
+    affiliation: 1
+  - name: Quentin Dessain
```
Comment (htwangtw) [inline paper/paper.md:16]: Please add ORCID and affiliation, thank you! @Hyedryn

File: paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  - BIDS
+  - Python
+  - fMRI
+  - functional connectivity
+authors:
+  - name: Hao-Ting Wang
+    corresponding: true
+    orcid: 0000-0003-4078-2038
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Rémi Gau
```
Comment (htwangtw) [inline paper/paper.md:13]: Please add ORCID and affiliation, thank you! @Remi-Gau

File: paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  - name: Natasha Clarke
+    affiliation: 1
```
Comment (htwangtw) [inline paper/paper.md:15]: Please add ORCID and affiliation, thank you! @clarkenj

File: paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  - name: Quentin Dessain
+  - name: Pierre Bellec
+    orcid: 0000-0002-9111-0699
+    affiliation: 1
+affiliations:
+- name: Centre de Recherche de l'Institut Universitaire de Gériatrie de Montréal, Université de Montréal, Montréal, QC, Canada
+  index: 1
```
Comment (Remi-Gau) [inline paper/paper.md:22]:
```suggestion
  index: 1
- name: Neuro Data Science ORIGAMI Laboratory, McConnell Brain Imaging Centre, Faculty of Medicine, McGill University, Montréal, Canada.
  index: 2
```

**Pull request merged**